docs/community/working-group.md

```diff
@@ -0,0 +1,189 @@
+# Working Groups
+
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
+
+The list, description, and membership of the existing Working Groups is available
+[here](working-group-definitions.md).
+
+## Terms
+
+* **Contributor** is used interchangeably within this doc as anyone participating in issues or
+  contributing code/RFCs/etc, regardless of their status with the PowerShell project.
+* **Repository Maintainers** are trusted stewards of the PowerShell repository responsible for
+  maintaining consistency and quality of PowerShell code.
+  One of their primary responsibilities is merging pull requests after all requirements have been fulfilled.
+  (Learn more about the Repository Maintainers [here](https://github.com/PowerShell/PowerShell/tree/master/docs/maintainers).)
+* The **PowerShell Committee** is responsible for the design and governance of the PowerShell project,
+  primarily by voting to accept or reject review-for-comment (RFC) documents.
+  (Learn more about the PowerShell Committee [here](https://github.com/PowerShell/PowerShell/blob/master/docs/community/governance.md#powershell-committee).)
+* A **Working Group** is a collection of people responsible for providing expertise on a specific
+  area of PowerShell in order to help establish consensus within the community and Committee.
+  The responsibilities of Working Groups are outlined below.
+  (Note: while some experts within Working Groups may have more specific expertise in a sub-topic
+  of the team,
+  the intent is that each team is holistically and collectively responsible for making decisions
+  within the larger topic space.)
+
+## Goals
+
+In designing the WG process, the Committee had a few goals:
+
+1. Increase the velocity of innovation without compromising the stability of PowerShell
+1. Reduce the time spent by contributors on writing/reviewing PRs and RFCs that are not feasible
+1. Increase the formal authority of subject matter experts (SMEs) inside and outside of Microsoft
+1. Decrease the volume of required technical discussions held by the Committee
+
```

Comment:
Should we spell out _how_ we believe this WG process will achieve these goals (and do we have a planned means to measure that)?